fe/pom.xml

```diff
@@ -661,7 +661,7 @@ under the License.
             <dependency>
                 <groupId>org.projectlombok</groupId>
                 <artifactId>lombok</artifactId>
-                <version>1.18.16</version>
+                <version>1.18.20</version>
```

Comment:
Why bump this version?

Reply:
to support [test null](https://github.com/apache/incubator-doris/blob/79fd117d60bd053d892f8aaa8c9a4554a4b35ed7/fe/fe-core/src/test/java/org/apache/doris/planner/QueryPlanTest.java#L1718),or build will fail